autonomy/analyse/abci/app_spec.py

```diff
@@ -57,6 +87,107 @@ class OutputFormats:  # pylint: disable=too-few-public-methods
         MERMAID = "mermaid"
         ALL = (JSON, YAML, MERMAID)
 
+        default_output_files = {
+            JSON: DEFAULT_FSM_SPEC_JSON,
+            YAML: DEFAULT_FSM_SPEC_YAML,
+            MERMAID: DEFAULT_FSM_SPEC_MERMAID,
+        }
+
+    @classmethod
+    def from_yaml(cls, file: Path) -> Dict:
+        """Load from yaml."""
+        with open_file(file, mode="r", encoding="utf-8") as fp:
+            data = yaml.safe_load(fp)
+
+        validate_fsm_spec(data)
+        return data
+
+    @classmethod
+    def from_json(cls, file: Path) -> Dict:
+        """Load from json."""
+        with open_file(file, mode="r", encoding="utf-8") as fp:
+            data = json.load(fp=fp)
+
+        validate_fsm_spec(data)
+        return data
+
+    @classmethod
+    def load(
+        cls,
+        file: Path,
+        spec_format: str = OutputFormats.YAML,
+    ) -> Dict:
+        """Load FSM specification."""
+
+        if spec_format == cls.OutputFormats.JSON:
+            data = cls.from_json(file)
+        elif spec_format == cls.OutputFormats.YAML:
+            data = cls.from_yaml(file)
+        else:
+            raise ValueError(f"Unrecognized input format {spec_format}.")
```

Comment:
@jmoreira-valory Is it possible to load an FSM specification from a mermaid file? If yes, it would be nice to support that. 

The error message also needs updating

Reply:
Currently only YAML and JSON files are supported. I suggest drop support for JSON (less code to maintain).

Reply:
@DavidMinarsch What do you think?